.gitignore

```diff
@@ -1,9 +1,12 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 node_modules
+
 # dependencies
 /node_modules
 /.pnp
 .pnp.js
+./package-lock.json
+package-lock.json
```

Comment:
Please remove `./package-lock.json` and `package-lock.json` from here. They are files which needs to be in our repo for builds to kick-in.